pkg/controllers/crdcompatibilityoperator/controller.go

```
@@ -0,0 +1,346 @@
+	deployment := &appsv1.Deployment{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      operandDeploymentName,
+			Namespace: r.namespace,
+		},
+	}
+
+	_, err := controllerutil.CreateOrPatch(ctx, r.client, deployment, func() error {
+		deployment.Spec.Replicas = &replicas
+		deployment.Spec.Selector = &metav1.LabelSelector{
+			MatchLabels: map[string]string{
+				"k8s-app": operandLabel,
```

golangci-lint (2.12.2)

[error] string `k8s-app` has 4 occurrences, make it a constant

(goconst)